doc/source/examples_source/00-basic/05-surface_traces_lic.py

```diff
@@ -63,42 +68,48 @@
 )
 session.show("image", width=800, height=600)
 
+
 ###############################################################################
 # Change Visual Attributes
-# ----------------------------------------------------------
+# ------------------------
 #  Modify the attributes of the Surface Restricted Traces to
-#  be visually closer to Flourescene or Titantiam Dioxide (experimental use)
+#  be visually closer to flourescene or titanium dioxide (experimental use)
+#
 # .. image:: /_static/05_srt_lic_2.png
 
 SRTpart.colorbyrgb = [0, 1, 0]
 SRTpart.OPAQUENESS = 0.25
 session.show("image", width=800, height=600)
 
+
 ###############################################################################
 # Try Line Integral Convolution (LIC) instead
-# ----------------------------------------------------------
-#  As we don't already have a near-surface, non-zero vector defined we need to create 'Offset' Variable.
+# -------------------------------------------
+#  As we do not already have a near-surface, non-zero vector defined we need to create 'Offset' Variable.
 #  Create Offset Variable for Value of Momentum at 2.e-5 distance into fluid domain
-#  Specify Offset Variable as the variable for LIC.
-#  Specify High Contrast and 1 length for the LIC
+#  Specify Offset Variable as the variable for LIC
+#  Specify High Contrast and 1 length for LIC
 #  Specify that we want to see LIC for the Shuttle Surface
+#
 # .. image:: /_static/05_srt_lic_3.png
 
 SRTpart.VISIBLE = False
-session.ensight.part.select_byname_begin("(CASE:Case 1)Shuttle")
-session.ensight.variables.evaluate("OffsetVar = OffsetVar(plist,Momentum,2e-05)")
+shuttle = session.ensight.objs.core.PARTS["Shuttle"][0]
+momentum = session.ensight.objs.core.VARIABLES["Momentum"][0]
+offset_var = session.ensight.utils.variables.calculator.offsetvar([shuttle], momentum, 2.0e-5)
```

Comment:
@randallfrank you can pass also a list of part names instead of the part object. The same for the variable. The code should be able to handle it. It makes the code even easier

Reply:
@mariostieriansys  I thought I tried that and ran into an issue.  I'll try it again.   Yes, I want to show this stuff off.  It is so much nicer.